multiqc/modules/kat/kat.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from KAT """
+import logging
+from collections import OrderedDict
+import json
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='KAT', anchor='kat',
+                                            href="https://github.com/TGAC/KAT",
+                                            info="is an toolkit for analysing sequencing data via its k-mer spectra.")
+
+        # Find and load any KAT dist analysis reports
+        self.kat_data = dict()
+        for c_file in self.find_log_files('kat'):
+            s_name = self.clean_s_name(c_file['s_name'][:-14], c_file['root'])
+            content = json.loads(c_file['f'])
+            self.kat_data[s_name] = self.parse_kat_report(content)
+
+        # Filter to strip out ignored sample names
+        self.kat_data = self.ignore_samples(self.kat_data)
+
+        if len(self.kat_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.kat_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.kat_data, 'multiqc_kat')
+
+        headers = OrderedDict()
+        headers["kmer_peaks"] = {
```

Comment:
Please set a different [colour scale](http://multiqc.info/docs/#table-colour-scales-1) for each column to make it easier to separate the columns.